csxtools/lmfit_models.py

```diff
@@ -0,0 +1,200 @@
+import numpy as np

```

Comment:
You should use numpy docstrings 

https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt